moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -87,13 +89,23 @@ servo:
   }
 
 ################################ GENERAL CONFIG #############################
-  enable_parameter_update: {
-    type: bool,
-    default_value: false,
-    description: "If true, continuous check for parameter update is enabled \
-                  Currently only the following parameters are updated:
-                  1. override_velocity_scaling_factor
-                  2. robot_link_command_frame"
+
+  thread_priority: {
+    type: int,
+    default_value: 40,
+    description: "This value is used when configuring the servo loop thread to use SCHED_FIFO scheduling \
+                  We use a slightly lower priority than the ros2_control default in order to reduce jitter \
+                  Reference: https://man7.org/linux/man-pages/man2/sched_setparam.2.html",
+    validation: {
+      gt_eq<>: 0
+    }
+  }
+
+  incoming_command_timeout: {
+    type: double,
+    default_value: 0.1,
+    description: "Commands will be discarded if it is older than the timeout.\
+                   Important because ROS may drop some messages."
```

Comment:
You could keep the same comment as before? Seems the old comment had more details.

Reply:
Those additional details do not apply to the new version.
It was about "filtered halt" since the old implementation would always publish a message every iteration.